internal/terraform/node_action.go

```diff
@@ -22,11 +21,7 @@ type GraphNodeConfigAction interface {
 // nodeExpandActionDeclaration represents an action config block in a configuration module,
 // which has not yet been expanded.
 type nodeExpandActionDeclaration struct {
-	Addr   addrs.ConfigAction
-	Config configs.Action
-
-	Schema           *providers.ActionSchema
-	ResolvedProvider addrs.AbsProviderConfig
+	*NodeAbstractAction
```

Comment:
Asking from a lack of knowledge: Since we embedded the abstract node, can't we remove all the methods that the abstract node implements from the expand node?